### ISSUE TITLE
index.columns is not iterable

### ISSUE THREAD

**Issue by singhpk89** [VemtoOrg/vemto2-issues#145]
**Error Message:**
index.columns is not iterable

**Stack:**
```
TypeError: index.columns is not iterable
    at Object.eval (eval at compileAsync (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:318:649), <anonymous>:114:27)
    at kae.compileAsync (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:318:675)
    at kae.compileAsyncWithErrorTreatment (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:318:290)
    at Hy.compile (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:364:14378)
    at Hy.compileWithImports (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:364:13975)
    at async Hge.generateCreationMigration (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:1155:144447)
    at async Hge.getContent (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:1155:144070)
    at async Hge.getData (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:1155:143157)
    at async gn (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:1269:14166)
    at async ie (file:///Applications/Vemto.app/Contents/Resources/app.asar/renderer/assets/index.ffee13a7.js:1269:11786)
```

**Template Name:**
```
(anonymous template)
```

**Template Error Line:**
```
93
```

**Template Section:**
```js
91                        <% } else { %>
92                            $table->index([
93                                <% for (let column of index.columns) { %>
94                                '<$ column $>',
95                                <% } %>
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.14
darwin 22.5.0
Locale: en-GB
App version: 2.0.14

**Comment by TiagoSilvaPereira** [comment]
@singhpk89 thanks for reporting. Would it be possible to give more details so that I can reproduce this error?

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.15
